config/settings.go

```diff
@@ -68,42 +50,34 @@ func apiBaseUrl() (string, error) {
 	case "":
 		return "https://api.yourbase.io", nil
 	default:
-		return "", fmt.Errorf("Unknown environment (%s) and no override in the config file or environment available", profile)
+		return "", fmt.Errorf("determine API URL: unknown environment %s and no configuration set", profile)
 	}
-
-	return "", fmt.Errorf("Unable to generate URL")
 }
 
-func ApiUrl(path string) (string, error) {
-	if !strings.HasPrefix(path, "/") {
-		path = fmt.Sprintf("/%s", path)
-	}
-
-	if baseUrl, err := apiBaseUrl(); err != nil {
-		return "", fmt.Errorf("Can't determine API URL: %v", err)
-	} else {
-		return fmt.Sprintf("%s%s", baseUrl, path), nil
+func APIURL(path string) (string, error) {
```

Comment:
This is unchanged in this PR, but a nice cleanup here could be to make all these `*URL` methods return `url.URL`s instead, for safety and sanity.

Reply:
Created ch-2692 to track.